.gitignore

```diff
@@ -13,3 +13,7 @@ webapp/dist
 webapp/node_modules
 !.vscode/README.md
 !sdkconfig.defaults*
+webapp/package-lock.json
+webapp/.env
+tools/.env
```

Comment:
The intent is good, but these are already ignored by [line 3: `.env`](https://github.com/VIPnytt/Frekvens/pull/198/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L3)